### PR TITLE
DM-26497: updated for ts_salobj 6.1

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/ATMCSSimulator/mcs_csc.py [hunk 1]
```diff
@@ -584,9 +580,14 @@ def set_event(self, evt_name, **kwargs):
             Event name (without the ``evt_`` prefix)
         **kwargs : `dict`
             Data for ``ControllerEvent.set``
+
+        Returns
+        -------
+        did_put : `bool`
```
Comment (mareuter) [inline python/lsst/ts/ATMCSSimulator/mcs_csc.py:586]: Please consider removing the variable and just declaring a type. The type only specification is what the NumPy docstrings documentation recommends.

**Review by mareuter — APPROVED** [review]
Looks good. One comment that you can do what you wish with.

File: python/lsst/ts/ATMCSSimulator/mcs_csc.py (hunk 1 above)
Comment (mareuter) [reply python/lsst/ts/ATMCSSimulator/mcs_csc.py:586]: I withdraw the above comment. It's OK as is.

**Comment by r-owen** [comment]
I figured out a simple way to retain backwards compatibility with ts_salobj 6 so I made that minor update.

**Pull request merged**